docs/UserGuide/Server/Cluster Setup.md

```diff
@@ -76,6 +76,55 @@ When both exist, the specified configuration item will overwrite the configurati
 > nohup cluster\target\cluster-0.11.0-SNAPSHOT2\sbin\start-node.bat  -internal_meta_port 9007 -internal_data_port 40014 -cluster_rpc_port 55562
 ```
 
+## One-click build script of n nodes and n-1 replicas pseudo-distributed.
+```
+#!/bin/bash  
+
+#config the number of node
+NODE_NUM=3
+
+export CLUSTER_JAR_PATH=./
+#CLUSTER_SNAPSHOT_NAME is something like cluster-0.11.0-SNAPSHOT
+CLUSTER_SNAPSHOT_NAME=$(find $CLUSTER_JAR_PATH -name "cluster-*-SNAPSHOT" | sed 's#.*/##')
```

Comment:
After 0.12 was released, the folder name will not contain SNAPSHOT, it may looks like `iotdb-cluster-0.12-bin.zip`, if the user wants to use this script to deploy the cluster,  it will not work.